std/container/binaryheap.d

```diff
@@ -125,11 +126,26 @@ if (isRandomAccessRange!(Store) || isRandomAccessRange!(typeof(Store.init[])))
     {
         assert(!store.empty, "Cannot pop an empty store.");
         if (store.length == 1) return;
-        auto t1 = store[].moveFront();
-        auto t2 = store[].moveBack();
+        static if(isRandomAccessRange!Store)
```

Comment:
> space after if. @CyberShadow do we catch that automatically yet?
> @wilzbach would know, ping

We do, but our automatic style checking got disabled this autumn ->  https://github.com/dlang/phobos/pull/4850